src/include/kdump/uifunctions.rb

```diff
@@ -1509,12 +1509,19 @@ def StoreInitrdKernel(_key, _event)
     # "Dump Format"
 
     def InitDumpFormat(_key)
-      if Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "ELF"
+      case Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT")
+      when "ELF"
         UI.ChangeWidget(Id("DumpFormat"), :Value, "elf_format")
-      elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "compressed"
+      when "compressed"
         UI.ChangeWidget(Id("DumpFormat"), :Value, "compressed_format")
-      elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "lzo"
+      when "lzo"
         UI.ChangeWidget(Id("DumpFormat"), :Value, "lzo_format")
+      when "snappy"
+        UI.ChangeWidget(Id("DumpFormat"), :Value, "snappy_format")
+      when "zstd"
```

Comment:
Are these formats supported by the SLE15-SP6 kernel?

Reply:
yes they are (by the SP6 makedumpile, it's a feature of makedumpfile, not the kernel)